include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -208,6 +208,24 @@ NOTE(iterator_method_unavailable, none, "C++ method '%0' that returns an "
 NOTE(iterator_potentially_unsafe, none, "C++ methods that return iterators "
                                         "are potentially unsafe; try using Swift collection APIs instead", ())
 
+ERROR(private_fileid_attr_repeated, none,
```

Comment:
This appears unused.

Reply:
Oops, thanks for catching that! I just checked in the code in `ImportDecl.cpp` that actually emits these diagnostics.